generators/cloud-init_test.go

```diff
@@ -111,19 +115,21 @@ local-hostname: {{ container.name }}
 		},
 		{
 			"vendor-data",
-			`{{ config_get("user.vendor-data", properties.default) }}
+			`{%- if config_get("cloud-init.user-data", properties.default) == properties.default -%}
+{{ config_get("user.user-data", properties.default) }}
+{%- else -%}
+{{- config_get("cloud-init.user-data", properties.default) }}
+{%- endif -%}
```

Comment:
Same as above, all of this is supposed to be `vendor-data`, not `user-data`.

Reply:
Fixed